SWLOR.Game.Server/Feature/QuestDefinition/HutlarQuestDefinition.cs

```diff
@@ -155,5 +156,42 @@ private void StupendousSlugBile()
                 .AddItemReward("slug_shake", 1);
         }
 
+        private void BreakTheByysk()
+        {
+            _builder.Create("break_the_byysk", "Break the Byysk")
+
+                .AddState()
+                .SetStateJournalText("Sharene wants you to kill One Hundred Byysk. Of you go!")
+                .AddKillObjective(NPCGroupType.Byysk_Guardian, 100)
+
+                .AddState()
+                .SetStateJournalText("Return to Sharene in the Hutlar Outpost and report your progress.")
+
+                .AddState()
+                .SetStateJournalText("Sharene wants you to kill another One Hundred Byysk. Better get going")
+                .AddKillObjective(NPCGroupType.Byysk_Guardian, 100)
+
+                .AddState()
+                .SetStateJournalText("Return to Sharene, once again, at the Hutlar Outpost.")
+
+                .AddState()
+                .SetStateJournalText("Sharene wants you to kill ANOTHER One Hundred Byysk...")
```

Comment:
I'm already annoyed that you're making the player go kill 400 Byysk and I'm not even the one playing. We need to rethink this quest.

Reply:
Droped this to a single stage two hundred and fifty Byssk